pkg/rancher-desktop/main/deploymentProfiles.ts

```diff
@@ -113,28 +117,79 @@ export function readDeploymentProfiles(): settings.DeploymentProfileType {
   return profiles;
 }
 
+/**
+ * Read and parse plutil deployment profile files.
+ * @param rootPath the system or user directory containing profiles.
+ * @param defaultsPath the file path to the 'defaults' file.
+ * @param lockedPath the file path to the 'locked' file.
+ * @returns the defaults and/or locked objects if they exist, or
+ *          throws an exception if there is an error parsing the locked file.
+ */
+async function parseJsonFromPlist(rootPath: string, defaultsPath: string, lockedPath: string) {
+  let defaults;
+  let locked;
+
+  const plutilArgArray = ['-convert', 'json', '-r', '-o', '-', '--'];
```

Comment:
I don't see what the first `'-'` is used for. The `'--'` indicates that all the subsequent arguments are filenames, but the `'-'` looks like a typo.